py/desisim/spec_qa/redshifts.py

```diff
@@ -257,12 +258,14 @@ def obj_requirements(zstats, objtype):
     -------
     dict
         Pass/fail dict
-    '''
+    """
     log = get_logger()
     pf_dict = {}
     #
     all_dict=dict(ELG={'RMS_DZ':0.0005, 'MEAN_DZ': 0.0002, 'CAT_RATE': 0.05, 'EFF': 0.90},
         LRG={'RMS_DZ':0.0005, 'MEAN_DZ': 0.0002, 'CAT_RATE': 0.05, 'EFF': 0.95},
+                  BGS={'RMS_DZ':1.0000, 'MEAN_DZ': 1.0000, 'CAT_RATE': 1.00, 'EFF': 0.0}, # MAKE BELIEVE
+                  MWS={'RMS_DZ':1.0000, 'MEAN_DZ': 1.0000, 'CAT_RATE': 1.00, 'EFF': 0.0}, # MAKE BELIEVE
```

Comment:
Even though we don't have formal requirements for BGS and MWS, let's pick something semi-reasonable so that some errors are triggered if they fall apart.  e.g. same as LRG, or maybe a bit better if we're feeling optimistic (current sims are so optimistic that no one believes them...)